code/datums/supplypacks.dm

```diff
@@ -462,6 +462,27 @@ WEAPONS
 	containername = "\improper explosive HEDP grenade crate (WARNING)"
 	group = "Weapons"
 
+/datum/supply_packs/explosives_hidp
+ 	name = "M40 HIDP incendiary grenade box crate (x25)"
```

Comment:
The problem may not be obvious at first, but my intuition was right - I used the "convert indentation to tabs" feature of Sublime Text and this is where a space is, at the beginning of the line.
![gitkraken_2019-01-11_14-01-22](https://user-images.githubusercontent.com/6252677/51035224-9251c700-15a9-11e9-81d2-5b6438885bbc.png)